server/py/services/api/api/endpoints/model_monitoring.py

```diff
@@ -327,3 +331,101 @@ def set_model_monitoring_credentials(
         stream_profile_name=stream_profile_name,
         replace_creds=replace_creds,
     )
+
+
+@dataclass
+class _FunctionSummariesParams:
+    project: str
+    auth_info: mlrun.common.schemas.AuthInfo
+    db_session: Session
+    start: datetime
+    end: datetime
+
+
+async def _common_function_parameters(
+    project: Annotated[
+        str,
+        Path(pattern=mlrun.common.schemas.model_monitoring.constants.PROJECT_PATTERN),
+    ],
+    auth_info: Annotated[
+        mlrun.common.schemas.AuthInfo, Depends(deps.authenticate_request)
+    ],
+    db_session: Annotated[Session, Depends(deps.get_db_session)],
+    client_version: Optional[str] = Header(
+        None, alias=mlrun.common.schemas.HeaderNames.client_version
+    ),
+    start: Optional[datetime] = None,
+    end: Optional[datetime] = None,
+) -> _FunctionSummariesParams:
+    """
+    Verify authorization and return common parameters.
+
+    :param project:         Project name.
+    :param auth_info:       The auth info of the request.
+    :param db_session:      A session that manages the current dialog with the database.
+    :returns:          A `_CommonParameters` object that contains the input data.
+    """
+
+    await _verify_authorization(
+        project=project,
+        auth_info=auth_info,
+        client_version=client_version,
+        action=mlrun.common.schemas.AuthorizationAction.read,
+    )
+    if start is None and end is None:
+        end = mlrun.utils.helpers.datetime_now()
+        start = end - timedelta(days=1)
+    elif start is not None and end is not None:
+        if start.tzinfo is None or end.tzinfo is None:
+            raise mlrun.errors.MLRunInvalidArgumentTypeError(
+                "Custom start and end times must contain the timezone."
+            )
+        if start > end:
+            raise mlrun.errors.MLRunInvalidArgumentError(
+                "The start time must precede the end time."
+            )
+    else:
+        raise mlrun.errors.MLRunInvalidArgumentError(
+            "Provided only one of start time, end time. Please provide both or neither."
+        )
+
+    return _FunctionSummariesParams(
+        project=project,
+        auth_info=auth_info,
+        db_session=db_session,
+        start=start,
+        end=end,
+    )
+
+
+@router.get("/function-summaries")
+async def get_model_monitoring_function_summaries(
+    commons: Annotated[_FunctionSummariesParams, Depends(_common_function_parameters)],
+    names: Optional[list[str]] = Query(None, alias="name"),
+    labels: list[str] = Query([], alias="label"),
+    include_stats: bool = Query(True, alias="include_stats"),
```

Comment:
Shouldn't the default be `False`?

Reply:
amm in my opinion, it makes more sense for the API (the main client of this API is the UI) to include the statistics by default. However, for the SDK, I believe it's better to keep this option disabled  by default, and only return the tsdb stats if the user explicitly requests them.
If we have to keep consistency here, I'll change it to `True` on the SDK side.